### PR TITLE
Add Azure Container Instances guidance 

### PR DESCRIPTION
**Pull request by jonathanalgar**

### CODE REVIEW

**Review by joaomvalentim — CHANGES_REQUESTED** [review]
We have agreed that we should not include specific container runtimes here like azure, and rather have technical docs with examples. So the Dockerfile change is ok, it makes the command clearer, but the doc part regarding ACI should not be included here.

**Review by OS-kiranmalsetty — APPROVED** [review]

**Comment by joaomvalentim** [comment]
This PR addresses two things:
- Dockerfile fix
- Documentation updates for Azure ACI.
Can we please split this to be able to approve only the first one?

File: README.md [hunk 1]
```diff
@@ -65,9 +66,13 @@ You may want to configure the binary to run as a service so it can start on syst
...
 
-Run the Docker image directly from the OutSystems GitHub Container registry:
+Run the Docker image directly from the OutSystems GitHub container registry:
 
-    docker run --rm -it ghcr.io/outsystems/outsystemscc:latest --help
+    docker run --rm -it ghcr.io/outsystems/outsystemscc --help
+
+If you're running the container on a runtime where you need to specify the command line or override the entrypoint (for example on Azure Container Instances or AWS Fargate):
+
+    docker run --rm -it --entrypoint /app/outsystems ghcr.io/outsystems/outsystemscc --help
```
Comment (bmartins-unit) [inline README.md:75]: Shouldn't be `--entrypoint /app/outsystemscc` ?
In PR: https://github.com/OutSystems/cloud-connector/pull/21/files the filename is `outsystemscc`

**Review by bmartins-unit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joaomvalentim) [reply README.md:75]: Correct, please fix also the doc. Thank you

**Review by bmartins-unit — APPROVED** [review]

**Review by bmartins-unit — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -141,64 +148,93 @@ If your organization uses a centralized log management product, see its document
...
+        The outsystemscc process is listening for:
+        a SIGUSR2 to print process stats, and
+        a SIGHUP to short-circuit the client reconnect timer
 
-    example remotes
+## 5. <a name="faqs"></a> FAQs <small><sup>[Top ▲](#table-of-contents)</sup></small>
 
-	  R:8081:192.168.0.3:8393
-    R:8082:192.168.0.4:587
+### How do I run `outsystemscc` on Azure Container Instances?
```
Comment (joaomvalentim) [inline README.md:209]: As agreed, this section should not be here. We have a generic comment above, so a dedicated ACI section is not needed and should be removed.

**Review by joaomvalentim — CHANGES_REQUESTED** [review]
Remove the ACI section

**Review by joaomvalentim — DISMISSED** [review]
lgtm

**Review by bmartins-unit — APPROVED** [review]

**Pull request merged**